content/en/contribute/code/releasing/publish-docker-image.md

```diff
@@ -8,15 +8,48 @@ relatedContent: >
     core/releases
 ---
 
-Docker images for CHT projects can be published to the [medicmobile](https://hub.docker.com/u/medicmobile) Docker Hub organization, so they are easily accessible to the community. This process can be automated using GitHub actions.
+## Setup Image Repository
+
+Docker images can be published to Docker Hub or Amazon ECR. (You should only publish to one of these repositories. There is no need to store the same image in both places.)
 
-## Create repository on Docker Hub
+### Configure repository on Docker Hub
+
+Docker images for CHT projects can be published to the [medicmobile](https://hub.docker.com/u/medicmobile) Docker Hub organization, so they are easily accessible to the community. This process can be automated using GitHub actions.
 
 First, create a repository for your new image on Docker Hub.
 
 1. Use the admin Docker account to [create a new repository](https://hub.docker.com/u/medicmobile) in the `medicmobile` organization.
 2. For your new repository, update the permissions to give the `developers` team the ability to `Read & Write`.  This will allow the GitHub action to push to the repository.
 
+### Configure repository on ECR
+
+There are two Amazon ERC repositories that can be used for publishing CHT images. `public.ecr.aws/medic` contains the released versions of the images, while `720541322708.dkr.ecr.eu-west-2.amazonaws.com/medic` is used to store branch builds (useful for CI testing, etc).
```

Comment:
while we may have multiple registries (the "R" in "ECR"), I don't think this split is correct.  For example, [here's the public URL](https://gallery.ecr.aws/medic/cht-couchdb) for `cht-couchdb` which shows build tabs from 3 days ago for things like `4.17.0-dependabot-npm-and-yarn-scripts-deploy-jsonpath-plus-10.3.0`. I think the pubic ECR holds both branches and GA images.

Looking in our private ECR registry - I'm confused! I see the same branches built there too (eg same [4.17-dependabot image](https://eu-west-2.console.aws.amazon.com/ecr/repositories/private/720541322708/medic/cht-api?region=eu-west-2))?

top is private bottom is public:

![image](https://github.com/user-attachments/assets/613d4512-6db4-424f-9961-d1700cf19999)

Reply:
Hmm maybe @Hareet can weigh in here on the difference between those two ECR URLs. :pray:   Maybe we don't need to mention either of them here... :shrug:

Reply:
There are two repositories because one contains pre-e2e images, and the other contains post-e2e test images. We publish the images to the private ECR after we run unit tests and we use those images to run e2e test over. If e2e tests pass, we tag and push them to the public ECR, to be available for installation.

Reply:
Thank you Diana!